src/comment.rs

```diff
@@ -160,6 +160,18 @@ pub(crate) fn is_last_comment_block(s: &str) -> bool {
     s.trim_end().ends_with("*/")
 }
 
+/// Returns true if the first line of the passed string starts with a comment
+/// that end on that line.
+fn is_first_comment_ends_on_first_line(s: &str) -> bool {
+    if s.starts_with("/*") {
+        s.lines().next().map_or(false, |l| l.contains("*/"))
```

Comment:
should we use `ends_with` instead of `contains`?
```suggestion
        s.lines().next().map_or(false, |l| l.ends_with("*/"))
```

Reply:
Changed (although I am not sure whether the comment was deleted).
The main benefit of the change is that line that ends with `/* comment 1*/ /* comment 2` is now properly handled.  Therefore I also changed the function name and its description.

While evaluating the suggested change I found two issues that are worth noting:

1. The following code is not handled properly and therefore is not formatted:
```rust
var= /* Block comment 1*/ // Line comment 2
value;
```
Fixing this is quite complex. Since it is not expected to be used much in practice, I assume that for this PR it is o.k. not to handle this case, but I did add `FIXME` comment for it.

2. the following code (now added to the test cases):
```rust
var = /* Block comment */
value;
```
Is formatted into one line:
```rust
var = /* Block comment */ value;
```
However:
```rust
var = /* Block comment single line 1 */ 
/* Block comment single line 2*/ value;
```
is formatted as:
```rust
var = /* Block comment single line 1 */ 
    /* Block comment single line 2*/
    value;
```
I believe this is o.k., but I note that in case the formatted `value` in the first example should be in the next line.